### PR TITLE
🎨 add breederdao skin

### PR DESCRIPTION
**Pull request by johnpaulcas**
#### Hi! What is your PR about?

- [x] Add or edit a skin
- [ ] Add a custom domain for your space
- [ ] Add an alias to migrate your space

### CODE REVIEW

**Review by gokaiorg — CHANGES_REQUESTED** [review]
To create your skin will also need to add your space name on index table: https://github.com/snapshot-labs/snapshot-spaces/blob/cf364b909692be68668a87701943bd65f217a6b0/skins/index.js#L302

**Comment by johnpaulcas** [comment]
> To create your skin will also need to add your space name on index table:
> 
> https://github.com/snapshot-labs/snapshot-spaces/blob/cf364b909692be68668a87701943bd65f217a6b0/skins/index.js#L302

hi @gokaiorg I guess its already added 👉 https://github.com/snapshot-labs/snapshot-spaces/pull/2444/files#diff-399e25f732c47d0534d12a9ab3cd620da0e32fcf3c9ef7f865e5b4c96f0e5dfb

**Comment by johnpaulcas** [comment]
> To create your skin will also need to add your space name on index table:
> 
> https://github.com/snapshot-labs/snapshot-spaces/blob/cf364b909692be68668a87701943bd65f217a6b0/skins/index.js#L302

hi @gokaiorg resolved this issue 👉 https://github.com/snapshot-labs/snapshot-spaces/pull/2444/files#diff-399e25f732c47d0534d12a9ab3cd620da0e32fcf3c9ef7f865e5b4c96f0e5dfbR340

thanks!

File: skins/index.js [hunk 1]
```diff
@@ -336,6 +337,7 @@ export default {
   bee2,
   benchmark,
   betswirl,
+  breederdaoSpace,
```
Comment (gokaiorg) [inline skins/index.js:340]: When you use a dash, you must use this naming convention.
```suggestion
  "breederdao-space": breederdaoSpace,
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: skins/index.js (hunk 1 above)
Comment (johnpaulcas) [reply skins/index.js]: > When you use a dash, you must use this naming convention.

resolved this issue @gokaiorg thanks!

**Review by sany12345 — COMMENTED** [review]
Good

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**